client/package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^12.1.10",
     "bootstrap": "^4.6.0",
     "react": "^17.0.1",
+    "react-bootstrap": "^1.5.0",
```

Comment:
react-bootstrap (which you have added here) and reactstrap (which was already added) are essentially the same thing done slightly differently. I am going to ask that we remove react-bootsrap and stick to reactstrap. This might require some refactoring in this PR